internal/clustercontroller/cluster/cluster.go

```diff
@@ -50,9 +50,9 @@ func (r *Reconciler) RegisterResources(ctx context.Context) error {
 	}
 	//rbac
 	sa := NewServiceAccountReconciler(*r.Client, clusterLables)
-	rb := NewClusterRoleBindingReconciler(*r.Client, clusterLables)
 	r.AddResource(sa)
-	r.AddResource(rb)
+	// rb := NewClusterRoleBindingReconciler(*r.Client, clusterLables)
+	// r.AddResource(rb)
```

Comment:
**Suggestion:** Consider checking for errors when adding resources to the reconciler. This ensures that any failures during the resource addition are handled appropriately. [possible issue, importance: 8]
```suggestion
 rb := NewClusterRoleBindingReconciler(*r.Client, clusterLables)
 if err := r.AddResource(rb); err != nil {
     return fmt.Errorf("failed to add resource: %w", err)
 }
```